### PR TITLE
Add links

### PR DESCRIPTION
**Pull request by luisDanielRoviraContreras**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,14 +23,13 @@ https://vuepress.vuejs.org/
 - [Built-in markdown extensions](https://vuepress.vuejs.org/guide/markdown.html) optimized for technical documentation
 - [Ability to leverage Vue inside markdown files](https://vuepress.vuejs.org/guide/using-vue.html)
 - [Vue-powered custom theme system](https://vuepress.vuejs.org/guide/custom-themes.html)
-- [Service Worker Integration](https://vuepress.vuejs.org/config/#serviceworker)
```
Comment (ulivz) [inline README.md]: Why removing them?

File: README.md [hunk 2]
```diff
@@ -4,6 +4,14 @@
   </a>
 </p>
 
+<div align="center">
+
+**[Web Site](https://vuepress.vuejs.org/) |
+[Latest Guide](https://vuepress.vuejs.org/guide/) |
+[GitHub](https://github.com/vuejs/vuepress/)**
```
Comment (ulivz) [inline README.md]: `Github` is duplicate.

Comment (luisDanielRoviraContreras) [reply README.md]: It is functional when the readme is used in another page as **NPM**

Comment (ulivz) [reply README.md]: Hmmm... npm has supported for that.

Comment (luisDanielRoviraContreras) [reply README.md]: So what do you think about adding other important links like Config Reference, Default Theme Config?

**Pull request closed without merging**